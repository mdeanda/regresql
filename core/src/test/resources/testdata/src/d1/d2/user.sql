select id, username
from user_account
order by id asc;
