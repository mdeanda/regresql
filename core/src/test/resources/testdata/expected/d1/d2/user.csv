"id","username"
"1","mdeanda"
"2","sodamnmad"
"3","fancyuser"
